Sources/URLRequest+APIAdapter.swift

```diff
@@ -53,9 +53,19 @@ extension URLRequest {
     }
 
     private mutating func setURLEncoded(parameters: HTTPParameters) {
+        let allowedCharacters = CharacterSet(charactersIn: "!*'();:@&=+$,/?%#[] ").inverted
+        var queryItems = [URLQueryItem]()
+        parameters.forEach { (key, value) in
```

Comment:
In APIKit we mostly use functional methods for transforming collections. I propose using `compactMap`.

```suggestion
        let queryItems: [URLQueryItem] = parameters.compactMap { (key, value) in
```